rust/agents/scraper/migration/src/m20230309_000001_create_table_domain.rs

```diff
@@ -321,7 +329,7 @@ const DOMAINS: &[RawDomain] = &[
     RawDomain {
         name: "test1",
         token: "ETH",
-        domain: 13371,
+        domain: 133711,
```

Comment:
how about some prefix rather than a suffix so we can apply it to all test chains?

Reply:
works for me